src/neo-vm/Debugger.cs

```diff
@@ -71,15 +73,15 @@ public VMState StepOut()
 
         public VMState StepOver()
         {
-            if (engine.State.HasFlag(VMState.HALT) || engine.State.HasFlag(VMState.FAULT))
+            if (engine.State == VMState.HALT || engine.State == VMState.FAULT)
                 return engine.State;
-            engine.State &= ~VMState.BREAK;
+            engine.State = VMState.NONE;
```

Comment:
In the part below, this was translated to:
```cs
            if (State == VMState.BREAK)
                State = VMState.NONE;
```

Reply:
The previous condition check the other states